test/examples/cypress/README.md

```diff
@@ -0,0 +1,36 @@
+# Cypress Tests
+
+This directory contains functional UI tests that execute using [Cypress](https://www.cypress.io/).
+
+## Setup
+
+```shell
+# bootstrap kibana from the project root
+yarn kbn bootstrap
+
+# build the plugins/assets that cypress will execute against
+# Note: you may wish to add the "--watch" argument to run the optimizer in watch mode
+node scripts/build_kibana_platform_plugins
+```
+
+## Running tests
+
+**Note:** in the lexicon of Cypress, the "Test Runner" is a desktop app for developing tests and is usually
+accessed with the `cypress open` command. Running Cypress tests in CI requires a "headless" runner that is
+usually accessed with the `cypress run` command. This directory contains test configurations for both.
```

Comment:
Out of curiosity, what's the startup time difference between these? we assume that headless mode is faster ("quickly checking regressions") but how much may be worth tracking over time.